dev/io.openliberty.data.internal_fat_jpa/fat/src/test/jakarta/data/jpa/DataJPATestCheckpoint.java

```diff
@@ -77,13 +74,4 @@ public static void setUp() throws Exception {
     public static void tearDown() throws Exception {
         server.stopServer();
     }
-
-    static void configureEnvVariable(LibertyServer server, Map<String, String> newEnv) throws Exception {
-        Properties serverEnvProperties = new Properties();
-        serverEnvProperties.putAll(newEnv);
-        File serverEnvFile = new File(server.getFileFromLibertyServerRoot("server.env").getAbsolutePath());
```

Comment:
I think I put a blank `server.env` in server root to accomplish this.  Can you remove that file in this PR?

Reply:
Done